src/test/java/org/fundacionjala/pivotal/runner/Runner.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.pivotal.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+import org.testng.annotations.DataProvider;
+
+/**
+ * Cucumber TestNG runner class.
+ */
+@CucumberOptions(
+        plugin = {"pretty"},
+        features = {"src/test/resources/features"},
+        glue = {"org.fundacionjala.pivotal"}
+)
+public final class Runner extends AbstractTestNGCucumberTests {
+    @Override
+    @DataProvider
```

Comment:
Please consider modifying the dataprovider parallel attribute:
```java
@DataProvider(parallel = true)
```

Reply:
I add it at last commit, could you review please?